gcc/linux-musl/build.sh

```diff
@@ -17,6 +17,11 @@ else
 	echo "PKG_VERSION: '$PKG_VERSION'"
 fi
 
+cd $BUILD_PREFIX/bin
+ln -s "$AR" ar
```

Comment:
I'm not sure what would such generic solution be, a script with these commands sourced wherever I added them?

TBH the most proper solution would be to fix Makefiles upstream (add such patches to the recipes) so that they use the `$(AR)` etc. wherever there's the `ar` etc. called directly.

Reply:
let's leave it as it is then